### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Alexstarrr**
resolve problems last week and attempt to use eslint

### CODE REVIEW

**Review by Alexstarrr — COMMENTED** [review]
I tried to use eslint today. Do you recommend me to use it at this point? Because there are many alerts coming from eslint, auto fix cannot fix them all. And what confused me most is this **''Expected linebreaks to be 'LF' but found 'CRLF'.eslintlinebreak-style''** 
I went through this [https://eslint.org/docs/rules/linebreak-style](https://eslint.org/docs/rules/linebreak-style) and several answers on stackoverflow. I don't know what's the best practice in this.

**Comment by agerlet** [comment]
> I tried to use eslint today. Do you recommend me to use it at this point? Because there are many alerts coming from eslint, auto fix cannot fix them all. And what confused me most is this `''Expected linebreaks to be 'LF' but found 'CRLF'.eslintlinebreak-style''`
> I went through (this)[https://eslint.org/docs/rules/linebreak-style] and several answers on stackoverflow. I don't know what's the best practice in this.

The smart thing Git is doing is it converts 'CRLF` to `LF` when you commit the code. You are confused because you cannot see them. They "both" NewLine on different systems. Please feel free to read through [this](https://en.wikipedia.org/wiki/Newline) doc from wiki. 
I will recommend you to understand more about eslint right now. If you want, we can talk about it on Thursday.

File: .gitignore [hunk 1]
```diff
@@ -18,6 +18,10 @@
 .env.test.local
 .env.production.local
 
+# cache
```
[the remark this answers is not in the document]
Comment (agerlet) [reply .gitignore:21]: Which problem?

[the remark this answers is not in the document]
Comment (Alexstarrr) [reply .gitignore:21]: always auto generating .eslintcache
[https://github.com/facebook/create-react-app/issues/10161](url)

[the remark this answers is not in the document]
Comment (agerlet) [reply .gitignore:21]: .gitignore is to ignore files and folders for git commit. Not blocking eslint to generate the cache files.

[the remark this answers is not in the document]
Comment (Alexstarrr) [reply .gitignore:21]: I didn't express myself clear, it still shows up in the git commit after I put it in .gitingore. As you can see from the current pull-request.

**Pull request merged**